azurerm/resource_arm_policy_assignment.go

```diff
@@ -57,6 +58,34 @@ func resourceArmPolicyAssignment() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"location": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				ForceNew:         true,
+				StateFunc:        azure.NormalizeLocation,
+				DiffSuppressFunc: azure.SuppressLocationDiff,
+			},
+
+			"identity_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(policy.None),
+					string(policy.SystemAssigned),
+				}, false),
+			},
+
+			"identity_principal_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"identity_tenant_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
in other resources we've put this in a sub-block - I think this'd make sense to do here too - could we update this? Here's [an example from the VM resource](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_virtual_machine.go#L80-L110) with [the expand](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_virtual_machine.go#L640-L643) and [flatten functions](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_virtual_machine.go#L728-L730) for reference

Reply:
👍